src/radical/entk/__init__.py

```diff
@@ -1,9 +1,10 @@
 # Version
-from radical.entk.version import version, __version__
+from .version import version, __version__, version_short, version_detail
+from .version import version_base, version_branch, sdist_name, sdist_path
 
-from radical.entk.pipeline.pipeline import Pipeline
-from radical.entk.stage.stage import Stage
-from radical.entk.task.task import Task
+from .pipeline.pipeline import Pipeline
+from .stage.stage       import Stage
+from .task.task         import Task
```

Comment:
These imports are relative, but the one in line 9 is not. Shouldn't it be relative for consistency. Although, I think `pylint` and `pep8` complain with relative imports

Reply:
Why do they complain?  
Yes, I will look at `states`.